web/src/components/nomination/NominateForm.vue

```diff
@@ -32,6 +32,9 @@
     <job-input
       v-model="form.job"
     />
+    <affiliation-input
```

Comment:
What about naming the component `secondary-affiliation` to be consistent with Airtable?

Reply:
Good point!